src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int height;
+    private int firstLeg;
+    private int secondLeg;
+
+    public IsoscelesTrapezoid(int firstLeg, int secondLeg, int height, Color color) {
+        super(color);
+        this.setFirstLeg();
+        this.setSecondLeg();
+        this.setHeight();
+    }
+
+    public int getHeight() {
+        return height;
+    }
+
+    public void setHeight() {
+        this.height = height;
+    }
+
+    public int getFirstLeg() {
+        return firstLeg;
+    }
+
+    public void setFirstLeg() {
+        this.firstLeg = firstLeg;
+    }
+
+    public int getSecondLeg() {
+        return secondLeg;
+    }
+
+    public void setSecondLeg() {
+        this.secondLeg = secondLeg;
+    }
```

Comment:
```suggestion
    public int getHeight() {
        return height;
    }

    public int getFirstLeg() {
        return firstLeg;
    }

    public int getSecondLeg() {
        return secondLeg;
    }
```

ці сеттери не роблять нічого корисного, давай приберемо їх(те саме для інших класів фігур)